src/testcafe-runner.js

```diff
@@ -152,18 +160,19 @@ async function runReporter ({ results, metrics, assetsPath, browserName, startTi
   }
 }
 
-async function run (runCfgPath, suiteName) {
+async function run (runCfgPath, suiteName, timeoutSec) {
   const cfg = await prepareConfiguration(runCfgPath, suiteName);
   if (!cfg) {
     return false;
   }
 
+  cfg.timeoutSec = timeoutSec;
```

Comment:
I would suggest making this `cfg.timeoutSec = process.env.TESTCAFE_RUNNER_TIMEOUT || timeoutSec`, and then trying a test with a very low timeout to test the exit codes.

Reply:
More people are voting for hardcoding this value. If we need it in the future we can easily add it and also add to the `saucectl`. I will add 30 mins limit to our doc.